scene/surface-placement/src/main/java/com/esri/samples/surface_placement/SurfacePlacementController.java

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright 2017 Esri.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.esri.samples.surface_placement;
+
+import java.util.Arrays;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.RadioButton;
+import javafx.scene.control.Slider;
+import javafx.scene.control.ToggleGroup;
+
+import com.esri.arcgisruntime.geometry.Geometry;
+import com.esri.arcgisruntime.geometry.GeometryType;
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.geometry.SpatialReferences;
+import com.esri.arcgisruntime.layers.ArcGISSceneLayer;
+import com.esri.arcgisruntime.mapping.ArcGISScene;
+import com.esri.arcgisruntime.mapping.ArcGISTiledElevationSource;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.Surface;
+import com.esri.arcgisruntime.mapping.view.Camera;
+import com.esri.arcgisruntime.mapping.view.Graphic;
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay;
+import com.esri.arcgisruntime.mapping.view.LayerSceneProperties;
+import com.esri.arcgisruntime.mapping.view.SceneView;
+import com.esri.arcgisruntime.symbology.SimpleMarkerSymbol;
+import com.esri.arcgisruntime.symbology.TextSymbol;
+
+public class SurfacePlacementController {
+
+  @FXML private SceneView sceneView;
+  @FXML private Slider zValueSlider;
+  @FXML private ToggleGroup toggleGroup;
+  @FXML private RadioButton drapedBillboardedRadioButton;
+  @FXML private RadioButton drapedFlatRadioButton;
+  private Surface surface;
+  private Point surfaceRelatedPoint;
+  private Point sceneRelatedPoint;
+  private GraphicsOverlay drapedBillboardedOverlay;
+  private GraphicsOverlay drapedFlatOverlay;
+  private GraphicsOverlay absoluteOverlay;
+  private GraphicsOverlay relativeOverlay;
+  private GraphicsOverlay relativeToSceneOverlay;
```

Comment:
I think most of these variables could be local variables instead of class members, so no need to define them here.

Reply:
Nice, another good side effect of adding the loop in. Much cleaner there now, thanks!